identifiers/country-mx.csv

```diff
@@ -1,4 +1,4 @@
-id,name
+id,name,validFrom,validTo

```

Comment:
```suggestion
id,name,validFrom,validThrough

```

Reply:
Maybe just run `./scripts/compile.py mx`

Reply:
Runned compile.py mx, that rewrite the file country-mx. If needs any changes, let me know.

![image](https://github.com/opencivicdata/ocd-division-ids/assets/51327625/6af10027-0e01-4eef-8165-d578fea3499b)